src/coord/src/client.rs

```diff
@@ -312,6 +313,33 @@ impl SessionClient {
         .await
     }
 
+    /// Executes a previously-bound portal. Incomplete timestamp errors on the
+    /// system timeline are retried.
+    pub async fn execute_retry(
```

Comment:
Would this function fit better in protocol.rs instead? Unclear.

Reply:
Here SGTM. Why not make this the default behavior of `execute` though?